src/pages/resultsView/ResultsViewURLWrapper.ts

```diff
@@ -67,9 +96,20 @@ export default class ResultsViewURLWrapper extends URLWrapper<
                 generic_assay_groups: { isSessionProp: false },
                 exclude_germline_mutations: { isSessionProp: false },
                 patient_enrichments: { isSessionProp: false },
+
                 comparison_subtab: { isSessionProp: false },
                 comparison_overlapStrategy: { isSessionProp: false },
 
+                // plots
+                plots_horz_selection: {
+                    isSessionProp: false,
```

Comment:
seems like this ought to be a sessionProp because the configuration of the plot is essential to shareability.